data/campaigns/Dead_Water/scenarios/09_The_Mage.cfg

```diff
@@ -69,6 +73,56 @@
             [objective]
                 description= _ "Find Caladon the mage"
                 condition=win
+                [show_if]
+                    [variable]
+                        name=Caladon_discovered
+                        not_equals=yes
+                    [/variable]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "Move a merfolk unit to Caladon’s shack and talk to him"
+                condition=win
+                [show_if]
+                    [variable]
+                        name=Caladon_discovered
+                        equals=yes
+                    [/variable]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "You can also visit Caladon with Keshan"
+                condition=win
+                [show_if]
+                    [and]
+                        [variable]
+                            name=Caladon_discovered
+                            equals=yes
+                        [/variable]
+                    [/and]
+                    [and]
+                        [have_unit]
+                            id=Keshan
+                        [/have_unit]
+                    [/and]
+                [/show_if]
+            [/objective]
+            [objective]
+                description= _ "You can also visit Caladon with Inky"
+                condition=win
+                [show_if]
+                    [and]
+                        [variable]
+                            name=Caladon_discovered
+                            equals=yes
+                        [/variable]
+                    [/and]
+                    [and]
+                        [have_unit]
+                            id=Inky
+                        [/have_unit]
+                    [/and]
```

Comment:
The and tags don't do anything, right? Why are they used?

Reply:
> The and tags don't do anything, right? Why are they used?

I assume, that I can delete the [and] tags around [have_unit], right? But I don't think, I can delete them around the check for bat race OR bat variation, am I right?

Cannot test this now. Maybe on Friday.